python/paddle/tensor/stat.py

```diff
@@ -288,11 +288,19 @@ def nanmedian(x, axis=None, keepdim=False, name=None):
             the output Tensor is the same as ``x`` except in the reduced
             dimensions(it is of size 1 in this case). Otherwise, the shape of
             the output Tensor is squeezed in ``axis`` . Default is False.
+        mode (str, optional): Whether to use mean or min operation to calculate
+            the nanmedian values when the input tensor has an even number of non-NaN elements
+            along the dimension ``axis``. Support 'avg' and 'min'. Default is 'avg'.
         name (str, optional): Name for the operation (optional, default is None).
             For more information, please refer to :ref:`api_guide_Name`.
 
     Returns:
-        Tensor, results of median along ``axis`` of ``x``. The output dtype is the same as `x`.
+        Tensor or tuple of Tensor.
+        If ``mode`` == 'avg', the result will be the tensor of nanmedian values;
+        If ``mode`` == 'min' and ``axis`` is not int (None or list or tuple), the result will
+            be the tensor of nanmedian values;
+        If ``mode`` == 'min' and ``axis`` is int, the result will be a tuple of two tensors
```

Comment:
这样写吧：

If ``mode`` == 'min' and ``axis`` is int, the result will be a tuple of two tensors (nanmedian value and nanmedian index). Otherwise, only nanmedian value will be returned.

看起来简介易懂点

Reply:
已修改